recipes/cmeutils/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "cmeutils" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cmelab/cmeutils/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: bc8ba46d485e35d45f9d0e719a139afaaacf534ae44e55c1ca1f7ac314671067
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - cython
```

Comment:
Is `cython` required? I don't see it in the source code. Also it seems like `pip` shouldn't be a run time requirement either.

Reply:
`cython` is not required, but the build for [27f2761](https://github.com/conda-forge/staged-recipes/pull/18608/commits/27f27618c51c16442ad10daabadbb5701ea77041) was failing because `freud` was not pulling `cython`.  I will make an issue on `freud` and remove `pip`.

Reply:
It should be safe to remove `cython` from the runtime requirements of this package.

Cython was mistakenly listed in `install_requires` for freud, meaning that `pip check` was searching for Cython as a runtime requirement. It is only a build-time requirement, which is correctly listed in the freud feedstock but not in freud's `setup.py`. This caused `pip check` to fail. This PR will fix the upstream for freud's next release: https://github.com/glotzerlab/freud/pull/938 In the meantime, I would skip the `pip check` test.